platform/android/java/editor/build.gradle

```diff
@@ -0,0 +1,53 @@
+// Gradle build config for Godot Engine's Android port.
+apply plugin: 'com.android.application'
+
+dependencies {
+    implementation libraries.kotlinStdLib
+    implementation libraries.androidxFragment
+    implementation project(":lib")
+}
+
+android {
+    compileSdkVersion versions.compileSdk
+    buildToolsVersion versions.buildTools
+    ndkVersion versions.ndkVersion
+
+    defaultConfig {
+        // The 'applicationId' suffix allows to install Godot 3.x(x3) and 4.x(x4) on the same device
+        applicationId "org.godotengine.editor.x3"
```

Comment:
Maybe `v3` and `v4` would be clearer? Assuming `3x` and `4x` were not possible due to restrictions on identifiers starting with a number?